### ISSUE TITLE
Scroll-Snap Support

### ISSUE THREAD

**Issue by markus-kraemer** [Grsmto/simplebar#437]
Is CSS scroll-snap supported with simplebar?

**Comment by Grsmto** [comment]
Just tested and yes it works just fine https://codesandbox.io/s/broken-cloud-lg3dc

**Comment by markus-kraemer** [comment]
Great. Horizontal too?

**Comment by strarsis** [comment]
Awesome! Would it even be compatible with a scroll-snap polyfill?

**Comment by strarsis** [comment]
@Grsmto, @markus-kraemer: Well, apparently dragging the scroll handle results in the behaviour of `scroll-snap-stop: always`, it isn't possible to drag items halfways into view. But it is possible to scroll smoothly using e.g. the mousewheel or keys (including scroll snapping).

**Comment by Grsmto** [comment]
That's right it seems like a limitation. I don't see an easy fix for this except very hacky solution, probably not worth the hassle to implement it in the library. 
Would be good to add this as a known limitation in the documentation.

**Comment by strarsis** [comment]
@Grsmto: So the issue is that the browsers don't fire the scroll events when scroll snapping is in progress?

**Comment by Grsmto** [comment]
@strarsis no, the issue si that SimpleBar does `el.scrollTop = x` to position the div scroll when dragging the scrollbar but for some reason css scroll snapping "blocks" the element from being in-between 2 snap points.
It seems like the native scrollbar doesn't just do `scrollTop` under the hood but does some more complex stuff to allow that it snaps only on mouse release and not while dragging around.
Unfortunately I tried to disable/enable the css snap property on drag start/stop but it doesn't work.

**Comment by strarsis** [comment]
@Grsmto: Yes, even `scrollTo` with animation parameter doesn't work, 
the scroll position changes instantaneously with scroll-snapping.
